github/data_source_github_repository_test.go

```diff
@@ -55,29 +55,35 @@ func TestAccGithubRepositoryDataSource(t *testing.T) {
 
```

Comment:
The name of this test should be changed if it's kept, maybe to something like `t.Run("does not error when a repository is not found", ...`.

I agree with @jcudit that I think the test should probably be removed. What do you think?

Reply:
Right now it doesn't make much sense, hehe. I'm reluctant to remove it entirely, though, since we're changing the functionality here. Give me a little more time to dig into Terraform testing and I'll see if I can come up with something to verify that this change works.